Corona-Warn-App/src/main/assets/privacy_en.html

```diff
@@ -1226,7 +1226,7 @@ <h2>
 </p>
 
 <p>
-    In addition, the confirmation of the authenticity of your app may involve the transfer of data
+    The EU has issued an adequacy decision for Switzerland, which determines the adequacy of the level of data protection in the country (Art. 45 GDPR). In addition, please note that users can retrieve the latest positive lists regardless of where they are (even if they are abroad on holiday or on a business trip, for example). In addition, the confirmation of the authenticity of your app may involve the transfer of data
```

Comment:
This now sounds rather repetitive because of "In addition, […] In additon, […]". May I suggest replacing one of the two with "Furthermore, […]"?